.ci-mgmt.yaml

```diff
@@ -0,0 +1,10 @@
+template: native
```

Comment:
I checked and it looks like `make ci-mgmt` isn't clean for this repo because you made some changes to the workflows after generating. I'd like us to do the little bit of additional work to generalize the templates so they work for this provider. We can't effectively maintain a provider without gettign it managed by ci-mgmt.

Reply:
Put together, these PRs update ci-mgmt so they generate the correct CI scripts for this provider:

- https://github.com/pulumi/ci-mgmt/pull/1493
- https://github.com/pulumi/ci-mgmt/pull/1492
- https://github.com/pulumi/ci-mgmt/pull/1491